test/serializer_test.exs

```diff
@@ -114,7 +112,7 @@ defmodule Membrane.IVF.SerializerTest do
 
   @tag :tmp_dir
   test "serialize real vp9 buffers", %{tmp_dir: tmp_dir} do
-    buffers = File.read!(Path.join(@fixtures_dir, "capture.dump")) |> :erlang.binary_to_term()
+    buffers = Path.join(@fixtures_dir, "capture.dump") |> File.read!() |> :erlang.binary_to_term()
```

Comment:
I cannot comment on line `132` of that file, but there are yet another some places where such a change with piping would be welcome ;)